app/factories/access_code.py

```diff
@@ -11,6 +11,7 @@ class AccessCodeFactory(factory.alchemy.SQLAlchemyModelFactory):
     class Meta:
         model = AccessCode
         sqlalchemy_session = db.session
+        sqlalchemy_session_persistence = 'commit'
```

Comment:
Why add this?

Reply:
sir, as created_at can't be assigned any value in a factory. we need to commit it in db to retrieve actually stored value in db so that is used to commit.

Reply:
Then commit manually. No need of this